src/version.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "0.0",
+  "sourceCodeUrl": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
+  "issueTrackerUrl": "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
+}
```

Comment:
shouldn't that better be stored in the frontend config?

Reply:
I'm unsure, it's nothing directly related to a config file. Therefore it's okay in my opinion to have this extra file.

Reply:
No. These are information that are built into the app. They are not requested via ajax.

Reply:
I thought about the uniformity of one config variable instead of two, but we most likely won't use that elsewhere again, so if both of you don't want this I'll close that

Reply:
This values aren't variables. They are constants. They shouldn't change in runtime. By importing the json webpack will use the literal values.